concepts/numbers/introduction.md

```diff
@@ -1,17 +1,6 @@
 # Introduction
 
-In Rust, there are two main types of numbers availble for use: Integers and Floating-Point Types.
-
-## Integers
-An Integer is a whole number without a fractional component. They come in a number of variants ranging from 8-bit to 128-bit. The more bits you use, the larger the number can be, however, this increases the memory used. Each variant can be Signed or Unsigned. A Signed number allows for negative values whereas Unsigned does not. You can declare an Integer like so:
-```rust
-// Declaring and assigning an integer with a size of 8 bits
-let myInt: u8 = 5;
-```
-
-## Floating-Point Types
-A floating-point number includes decimal points and there are two types available in Rust. An `f32` and `f64` - both referring to a float of 32 bits and a float of 64 bits. `f64` is the default type in Rust as working with either size can be done at about the same speed when using a modern CPU. You can declare a Floating-Point Type like so:
+You can initialize numbers in Rust quickly and easily like this:
```

Comment:
I love the brevity! My other initial reaction is that I feel we should briefly summarize that Rust has a few numeric types first, then give concrete examples. 
So doing what you have here, but naming the specific numeric type in the syntax example.
And the first line summarizing the various numeric types Rust has.
Give it a whirl and let me know!

Reply:
That's fair! and a good point. It seems like numbers in rust fall into a couple of broad categories based on what you might need. 

When choosing what type of number to use in Rust two of the biggest questions to me are: 
- does this number need to support decimals?
- does this number need to be negative?

The size of the number is also important but probably not so much for an introduction. Maybe the signed / unsigned distinction is also not so important for an intro? Not sure, but trying to hone in on what kind of questions the intro can answer.

Also, I thought there might be some value in showing what types Rust defaults to if a type isn't specified. I did this through a syntax example to try and demonstrate how the compiler might make decisions for you if you don't explicitly set a type, although this isn't stated outright.

Reply:
Thanks for sharing your reasoning @sm1215! The content length you updated feels perfect to me.

> Also, I thought there might be some value in showing what types Rust defaults to if a type isn't specified. 

I like pointing out type inference at work. Seems helpful for introducing students to how the language works.

Small typo fix and then 👍 from me!